### PR TITLE
Prevent fixtures command from always returning 1

### PR DESCRIPTION
**Pull request by hacfi**
I was irritated why the fixtures seemed to fail.

### CODE REVIEW

File: src/Elcodi/Bundle/FixturesBoosterBundle/Command/LoadDataFixturesDoctrineCommand.php [hunk 1]
```diff
@@ -83,13 +83,11 @@ public function __construct(
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         if (empty($this->databaseFilePath)) {
-            parent::execute($input, $output);
-
-            return 1;
+            return parent::execute($input, $output);
```
Comment (xphere) [inline src/Elcodi/Bundle/FixturesBoosterBundle/Command/LoadDataFixturesDoctrineCommand.php]: So, that's why my refresh script was failing without errors after fixture loading. Thanks for the fix!
Just a minor thing, CS requires new line before return statements.

Comment (hacfi) [inline src/Elcodi/Bundle/FixturesBoosterBundle/Command/LoadDataFixturesDoctrineCommand.php]: Even if the code is indented? Ok..will update the PR

**Comment by alch** [comment]
@hacfi :+1: This problem was bugging us too

**Comment by xphere** [comment]
Merged in https://github.com/elcodi/elcodi/commit/a9176cc19d8d39062d963ff37c6564b8647ad848
Thanks @hacfi

**Pull request closed without merging**